Source/Android/jni/MainAndroid.cpp

```diff
@@ -564,13 +568,26 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_SurfaceDestr
     s_surf = nullptr;
   }
 }
+
+JNIEXPORT jfloat JNICALL
+Java_org_dolphinemu_dolphinemu_NativeLibrary_GetGameAspectRatio(JNIEnv* env, jobject obj)
+{
+  return g_renderer->CalculateDrawAspectRatio();
```

Comment:
This should really only be called by the video thread. But I can't think of a clean way around it off the top of my head.